platforms/ide/ide-plugins/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("gradlebuild.distribution.api-java")
+}
+
+description = "Plugins that add support for generating IDE project files used for importing Gradle projects into IDEs"
```

Comment:
Should this include something about the tooling model builder stuff?

Reply:
It probably should.  But after reviewing this again, I'm now once again not sure these classes belong here.  They are needed to avoid applying plugins by name instead of by class.

I'd like to hear the IDE team's opinion on where they belong.